test/servinge2e/servicemesh_test.go

```diff
@@ -638,65 +638,62 @@ func TestKsvcWithServiceMeshJWTDefaultPolicy(t *testing.T) {
 				"iat": time.Now().Unix(),
 				"exp": time.Now().Unix() + 3600,
 			},
-		},
-			{
-				// No token (request will be done without the Authorization header)
-				"no_token",
-				false,
-				nil,
-				nil,
+		}, {
```

Comment:
While we're at it, can we please include the keys in the respective tests too?

Reply:
Alright! Done